source/databricks/tests/conftest.py

```diff
@@ -19,14 +19,25 @@
 import pytest
 from pyspark import SparkConf
 from pyspark.sql import SparkSession
+from pyspark.sql.types import StringType
+from datetime import datetime
 
 
 # Create Spark Conf/Session.
 @pytest.fixture(scope="session")
 def spark():
-    spark_conf = (
-        SparkConf(loadDefaults=True)
-        .set("spark.sql.session.timeZone", "UTC")
-    )
+    spark_conf = SparkConf(loadDefaults=True).set("spark.sql.session.timeZone", "UTC")
 
     return SparkSession.builder.config(conf=spark_conf).getOrCreate()
+
+
+# Creates timestamp from utc string in correct format yyyy-mm-ddThh:mm:ss.nnnZ
+@pytest.fixture(scope="session")
+def timestamp():
```

Comment:
1. Let's add the comment the python way so we get intellisense (https://www.pythontutorial.net/python-basics/python-function-docstrings/)
2. When injecting the fixture `timestamp` is misleading. Let's call it a factory.

```suggestion
@pytest.fixture(scope="session")
def timestamp_factory():
    "Creates timestamp from utc string in correct format yyyy-mm-ddThh:mm:ss.nnnZ"
```

Reply:
Good idea. Have implemented this